core/trino-main/src/main/java/io/trino/security/AccessControlManager.java

```diff
@@ -176,7 +176,7 @@ public void loadSystemAccessControl()
 
     private SystemAccessControl createSystemAccessControl(File configFile)
     {
-        log.info("-- Loading system access control %s --", configFile);
+        log.debug("-- Loading system access control %s --", configFile);
```

Comment:
This could have been useful when troubleshooting. 
Generally the idea to log system configuration at startup is not a useless. This is probably why we use quiet bootstrap mode in tests only, right? The chosen access control is part of the system configuration, and should follow the same logic.

Reply:
See above: https://github.com/trinodb/trino/pull/19554#issuecomment-1790828066